backend/amd64/proc.ml

```diff
@@ -332,7 +332,7 @@ let destroyed_at_oper = function
       [| loc_spacetime_node_hole |]
   | Iswitch(_, _) -> [| rax; rdx |]
   | Itrywith _ -> [| r11 |]
-  | Iop(Ispecific (Irdtsc | Irdpmc)) -> [| rax |]
+  | Iop(Ispecific (Irdtsc | Irdpmc)) -> [| rdx |]
   | _ ->
```

Comment:
(Not related to this PR, but if we plan on adding more `Ispecific`
instructions, it might be safer to replace this catch-all with an
enumeration of the other `Ispecific` values.)

Reply:
Yes, it's a good idea, I will make a separate PR.